src/app/components/Onboard/Shared/card.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function Card({ topLabel, lastLabel, heading }) {
```

Comment:
Hey @namxam so this component is from the [wallet-ui-kit](https://www.figma.com/community/file/916680391812923706) and while it may only be used in this application once (so far) since the designs are based on the wallet-ui-kit @bumi and I spoke about being able to extract them out so other projects can use them.